Lib/test/test__xxsubinterpreters.py

```diff
@@ -1203,6 +1203,58 @@ def test_ids_global(self):
 
         self.assertEqual(cid2, int(cid1) + 1)
 
+    def _assert_interpreters_returned(self, cid, send, recv):
```

Comment:
FWIW, when it comes to test I prefer duplicating code if it means the test is easier to understand.  This helper is border-line doing enough that it obscures what the actual tests are all about.  But it isn't a huge problem, so I'll leave it to you.